docs/src/guides/drupal9/redis.md

```diff
@@ -1,149 +1,155 @@
 ---
 title: "Using Redis with Drupal 9.x"
 sidebarTitle: "Redis"
-description: |
-    Add Redis caching to your existing Drupal site.
+description: Add Redis caching to your existing Drupal site.
 weight: -70
 ---
 
-If you are using the Platform.sh-provided Drupal template, most of this work is already done for you.
-Redis is already configured and is enabled after the installation is complete.
+Redis is a fast open-source in-memory database and cache, 
+useful for application-level caching. 
+For more information on this service, see the [dedicated Redis page](../../add-services/redis.md) 
+or the [official Redis documentation](https://redis.io/docs/).
 
-Note that this `redis` service is ephemeral, meaning it doesn't persist if the container moves or is shut down.
-Your app must treat it as ephemeral and not rely on it being there.
+Follow the instructions on this page to do one of the following:
 
-There are two possible solutions:
+- Add and configure Redis for Drupal 9.x if you have deployed Drupal manually.
+- Fine-tune your existing configuration if you have deployed Drupal 9 using a [Platform.sh template](../../development/templates.md).
 
-* Change the service to persistent Redis (`redis-persistent`).
-* Clean the cache each time your app starts in the `start` key in [your web configuration](../../create-apps/app-reference.md#web-commands).
+## Before you begin
 
-If you are working from an older repository or migrating a pre-built site to Platform.sh, see the instructions below.
+You need:
 
-## Requirements
+- A [Drupal 9 version deployed on Platform.sh](../drupal9/deploy/_index.md)
+- The [Platform.sh CLI](../../administration/cli/)
+- [Composer](https://getcomposer.org/), which is recommended by both Platform.sh and Drupal 
+  to [add the Redis module to your Drupal project](../../guides/drupal9/redis.md#3-add-the-drupal-module).
+  Platform.sh also recommends you use Composer to manage your whole site and its dependencies
```

Comment:
:clipboard: Ideally, the list would be changed so it's all fragments and no sentences and in parallel, as [recommended in the style guide](https://developers.google.com/style/lists#numbered-lettered-bulleted-lists). But at least this:
```suggestion
  Platform.sh also recommends you use Composer to manage your whole site and its dependencies.
```